docs/index.rst

```diff
@@ -666,6 +680,12 @@ You can customize the instantiator in several ways:
         })
     ;
 
+.. warning::
+
+    The ``instantiateWith()`` method completely overrides the default instantiation
+    and object hydration system. Any attributes defined in the ``defaults()``
+    method will be ignored.
```

Comment:
Hmm, they aren't ignored, they're made available in the `array $attributes` callable argument. Of course they could have been overridden by `->with()` also.